TestAssets/CrossPublishTestProjects/StandaloneAppCrossPublish/project.json

```diff
@@ -0,0 +1,35 @@
+{
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
+      "dependencies": {
+        "Microsoft.NETCore.App": "1.0.0-rc2-*"
+      }
+    }
+  },
+  "runtimes": {
+    "win7-x64": {},
+    "win7-x86": {},
+    "osx.10.10-x64": {},
+    "osx.10.11-x64": {},
+    "ubuntu.14.04-x64": {},
+    "centos.7-x64": {},
+    "rhel.7.2-x64": {},
+    "debian.8.2-x64": {}
+  },
+
+  "runtimeOptions": {
+    "somethingString": "anything",
```

Comment:
As far as I know this is the current Plan of Record. 

If this is changing in the immediate term, this whole PR is not going to work because it depends on a host being present in the dependency graph for X-Publish.